### ISSUE TITLE
Inconsistend rendering of callout / codeblock admonitions

### ISSUE THREAD

**Issue by nousernameavailableanymore** [ceciliamay/obsidianmd-theme-primary#102]
Callout admonitions are rendered differently (I assume incorrectly) than codeblock admonitions.
In default theme, both are rendered identically.

Callout admonitions:
1. The drop shadow is rendered very strangely. It has sharp edges in the lower right (compared to the smooth edges of the codeblock admonitions).
2. There seems to be a (rather ugly) 1px frame around it.
3. The title bar color is incorrect (compared to the color that was configured in the admonition settings for this type).

![image](https://user-images.githubusercontent.com/45376324/160512408-4a64e8a5-4c3f-4446-9749-780529a25a90.png)

**Comment by ceciliamay** [comment]
i have no plans to support the admonitions plugin. if you do not like the design that primary offers for callouts, you are free to edit it with your own snippet. the colors don't precisely match because i have reduced the brightness (in light mode) and increased the brightness (in dark mode) of the icons so that it will always have good contrast ratio. as said, if you do not like it you are free to edit it with your own snippet.

based on the admonitions plugin, they have dropped support for msdocs syntax so i assume those are Obsidian native callouts. while i am aware that there are sharp cut offs of the shadows at the end, it is microscopic that it isn't even visible unless inspected. the shadows also do not render harshly like yours on my end:
![image](https://user-images.githubusercontent.com/90643719/160514671-9e304533-bdcf-423a-acb3-1a3991dc3bb1.png)

i might be missing something here or misunderstanding, please let me know! i do suggest to take a look at your snippets/update admonitions plugin.

**Comment by nousernameavailableanymore** [comment]
What does "i have no plans to support the admonitions plugin" exactly mean?
The native callouts now use the presets configured in the admonition plugin (icon, color...).

And everything is working fine in Primary with codeblock admonitions.
However, Primary seems to overwrite the defaults theme behavior in ignoring the settings in the now somewhat official admonition plugin.

Usually the Theme expands on the features that are available in the base theme.
But at the moment, Primary is taking a functionality away, that the default theme has (support for editing callout blocks).

Basically I m not asking to support anything (as it already works out of the box) but rather NOT to overwrite / interfere with a working implementation.
Just render callout blocks the same way as admonition blocks.

**Comment by ceciliamay** [comment]
you just answered your own question but it works the other way around. the admonition plugin (icon, colors, etc.) supports the native callouts. admonition adds functionality and style to the native callouts. 

so whatever by product design that the admonition plugin renders (including the harsh shadows) - i do not plan to have any support/tweak. there are no "native settings" for obsidian callouts. i am not taking any functionality away, it works the way it should. you have a callout block rendered. it's collapsible if you want to. you can even style them with your own **snippets**, using the customization css obsidian has given (not through admonitions plugin - that would insinuate "plugin support"). you just have an opinionated take on what the design should be but do not equate to my vision of the theme. 

the theme did what a theme might be expected to do: skin. again, if you want callout blocks to render the same as admonition blocks, create your own snippet. 

furthermore, it seems that admonition is adding bigger shadows to the native callouts. that is not my problem - because i do not support the plugin.

**Comment by nousernameavailableanymore** [comment]
I just did some additional testing and reverted back to version **1.3.3** (the one before the custom callouts).

Turns out, in this version, the callouts were rendered perfectly fine / correct.
So Primary DID actually (implicitly) support admonitions.

This correct rendering then broke in the next version.
I assume because ALL callouts are caught (instead of just the custom callouts for Primary), which then conflicts with the already present settings of the admonition plugin (the shadow seems to get combined from Primary + Admonition Plugin).

So everything worked fine up and including 1.3.3 and then broke (or to be exact "callout types that are created in admonition plugin are rendered incorrectly") in the next version.

Wouldnt it be possible to just style the Primary Theme specific callouts and leave all other callouts untouched?
This would not be "support for admonition plugin" (as this was already present in 1.3.3) but rather only styling the things that Primary adds.
